packages/vuetify/src/components/VStepper/VStepperStep.tsx

```diff
@@ -0,0 +1,151 @@
+// Styles
+import './VStepperStep.sass'
+
+// Components
+import { VAvatar } from '@/components/VAvatar'
+import { VBtn } from '@/components/VBtn'
+
+// Composable
+import { makeGroupItemProps, useGroupItem } from '@/composables/group'
+import { genericComponent } from '@/util'
+
+// Utilities
+import { computed, inject } from 'vue'
+import { VStepperGroupProvideSymbol, VStepperProvideSymbol } from './VStepper'
+
+// Types
+import { useTextColor } from '@/composables/color'
+
+export const VStepperStep = genericComponent()({
+
+  name: 'VStepperStep',
+
+  // inheritAttrs: false,
+
+  props: {
+    color: {
+      type: String,
+      default: 'primary',
+    },
+    complete: Boolean,
+    completeIcon: {
+      type: String,
+      default: '$complete',
+    },
+    editable: {
+      type: Boolean,
+      default: false,
```

Comment:
No need to set defaults for primitives. We use the intrinsic default values from Vue